views/index.handlebars

```diff
@@ -0,0 +1,155 @@
+<div class="row">
+  <div class="col-md-12">
+    {{{nav}}}
+    <p><a href="/doc">Publication rules</a></p>
+  </div>
+</div>
+
+<div id="about" class="alert alert-warning alert-dismissible" role="alert">
+  <button type="button" class="close" onclick="$('#infoLink').show(); $('#about').hide();"><span aria-hidden="true">&times;</span>
+    <span class="sr-only">Close</span>
+  </button>
+  This alternative pubrules checker is <strong>experimental</strong>, and provided only for early testing. <br>
+  Continue using the <a href="http://www.w3.org/2005/07/pubrules">official pubrules checker</a> for publishing;
+  only that one is considered authoritative.<br>
+  Contribute and submit issues <a href="https://github.com/w3c/specberus">on GitHub</a>; send comments to
+  <a href="mailto:public-pubrules-comments@w3.org"><code>public-pubrules-comments@w3.org</code></a>.
+</div>
+
+<div class="row">
+  <div class="col-md-12">
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">Test a document against the <a href="/doc">publication rules</a></h3>
+      </div>
+      <div class="panel-body">
+        <form id="options" class="form-horizontal manual">
+
+          <div class="form-group">
+            <label for="url" class="col-xs-3 col-sm-1 col-md-1 col-lg-1 control-label">URL</label>
+            <div id="urlContainer" class="col-xs-9 col-sm-4 col-md-5 col-lg-5">
+              <input type="url" class="form-control" id="url" required autofocus="autofocus" value="https://">
+            </div>
+            <label for="profile" class="col-xs-3 col-sm-1 col-md-1 col-lg-1 control-label">Profile</label>
+            <div id="profileContainer" class="col-xs-9 col-sm-4 col-md-4 col-lg-4">
+              <select class="form-control" id="profile" required>
+                <option value disabled="disabled">Select an option</option>
+              </select>
+            </div>
+            <div id="check1" class="col-sm-2 col-md-1 col-lg-1">
+              <button type="submit" class="btn btn-primary pull-right">Check</button>
+            </div>
+          </div>
+
+          <div class="form-group manual">
+            <label class="col-xs-3 col-md-1 col-lg-1 control-label"><abbr title="Patent Policy">PP</abbr></label>
```

Comment:
I suggest `s/PP/Patent Policy/` and remove `<abbr>` to have a clear interface. PP is not an acronym people are familiar with outside the team.